src/types/dataModel/volunteer.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod';
+import zOrganization from './organization';
+
+export const roles = ['Medical', 'Dental', 'Food', 'Save the Babies'] as const;
+export const zRole = z.enum(roles);
+export type Role = z.infer<typeof zRole>;
+
+export const backgroundCheckStatuses = [
+  'Passed',
+  'Failed',
+  'In Progress',
+  'Not Checked',
+] as const;
+export const zBackgroundCheckStatus = z.enum(backgroundCheckStatuses);
+
+const zVolunteer = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  email: z.string().email(),
+  phoneNumber: z.string(),
+  previousRole: zRole,
+  previousOrganization: zOrganization.optional(),
+  backgroundCheck: z
+    .object({
+      status: zBackgroundCheckStatus,
+      lastUpdated: z.date(),
+    })
+    .optional(),
```

Comment:
I think it makes sense to **either** have `volunteer.backgroundCheck` be optional **or** drop the "Not Checked" status. That way there is only one way to communicate that someone hasn't had a background check yet. I'd lean towards the latter option, it is a bit more explicit

Reply:
ill remove the not checked